packages/devextreme/testing/tests/DevExpress.core/utils.object.tests.js

```diff
@@ -171,7 +171,21 @@ QUnit.test('deepExtendArraySafe utility does not throw an error with \'null\' de
     assert.equal(result.deepProp.toChange, 'changed value');
 });
 
-QUnit.test('deepExtendArraySafe utility does not pollute object prototype', function(assert) {
-    objectUtils.deepExtendArraySafe({ }, JSON.parse('{ "__proto__": { "pollution": true }}'), true);
-    assert.ok(!('pollution' in { }), 'object prototype is not polluted');
+QUnit.test('deepExtendArraySafe sets undefined', function(assert) {
+    const objWithValue = { time: { duration: 50 } };
+    const objNoValue = {};
+    objectUtils.deepExtendArraySafe(objWithValue, { time: { duration: undefined } }, true);
+    objectUtils.deepExtendArraySafe(objNoValue, { time: { duration: undefined } }, true);
```

Comment:
Let's set all argument values in this test
It looks a little confusing now because instead of boolean flags we pass undefined :)